### PR TITLE
updating CC*, requirements, Additional Materials

### PR DESCRIPTION
**Pull request by lmichael107**
with new presentations and YouTube videos from the Oct 2020 campus workshop

### CODE REVIEW

**Comment by lmichael107** [comment]
@brianhlin please review

File: docs/policy/campus-cyberinfrastructure.md
Comment (brianhlin) [inline docs/policy/campus-cyberinfrastructure.md:24]:
```suggestion
and enable users to take advantage of these from submission points (whether via an OSG-operated submission point, like 
```

File: docs/policy/campus-cyberinfrastructure.md
Comment (brianhlin) [inline docs/policy/campus-cyberinfrastructure.md:57]: @lmichael107 Maybe we should store the slides in the repo as before? Probably not as strictly necessary as before with them being in Indico instead of Google Drive

File: docs/policy/campus-cyberinfrastructure.md
Comment (brianhlin) [inline docs/policy/campus-cyberinfrastructure.md:25]:
```suggestion
[OSG Connect](https://www.osgconnect.net/), or a locally-managed one).
```

File: docs/policy/campus-cyberinfrastructure.md
Comment (lmichael107) [reply docs/policy/campus-cyberinfrastructure.md:57]: yeah, let's keep it all in one place (Indico), which we just didn't have before.

**Comment by lmichael107** [comment]
Looks like there are some random failures of external links (also at least one instance where everything succeeded). Anything to do before merging? -L

**Review by osg-cat — APPROVED** [review]
I fixed a minor typo. The link that failed in the previous check now succeeded, and now there is a new link failure on another unrelated page. I think this change is fine to merge, assuming the content is good, etc.

**Pull request merged**
